src/aditiGJUN20/MeetingRoom.java

```diff
@@ -0,0 +1,134 @@
+/*
+Program to demonstrate overloading behavior.
+
+Create a class MeetingRoom having method bookMeetingRoom which should provide
+below facility:
+meeting room names: Nalanda, Takshashila, AgraFort, and PratapGadh
+
+a. If any user call method bookMeetingRoom() at that time“Nalanda” meeting room will be booked for 1 hr.
+b. Write a method that helps to book a meeting room other than "Nalanda" for default timing would be 1 hr.
+c. Write a method which helps to book a meeting room by giving the name of the meeting room and given hrs.
+d. displayInfo() method should print booked meeting room info [i.e. meeting room,
+booked for how many hrs].
+e. displayAvilableMeetingRoomForBooking() method will print available meeting room names. 
+     [i.e. If Nalanda and Takshashila is booked then this method should print "Available meeting rooms: AgraFort and 
+     PratapGadh"] 
+*/
+
+package aditiGJUN20;
+
+public class MeetingRoom {
+
+	enum MEETINGROOM {
+		NALANDA, TAKSHSHILA, AGRAFORT, PRATAPGADH
+	}
+
+	static int bookedHoursNalanda = 0, bookedHoursTakshashila = 0, bookedHoursAgraFort = 0, bookedHoursPratapGadh = 0;
+	static boolean nalandaBookingStatus, takshshilaBookingStatus, agraFortBookingStatus, pratapGadhBookingStatus;
+
+	void bookMeetingRoom() {
+		if (!nalandaBookingStatus) {
+			System.out.println("Nalanda Meeting room is booked for 1 hr");
+			nalandaBookingStatus = true;
+			bookedHoursNalanda++;
```

Comment:
here one suggestion ,as per the code when the status is true next time it will never pass the If condition. So the bookedHoursNalanda need to be mentioned as 1 hr instead of incrementing it.